bazel/python_rules.bzl

```diff
@@ -297,7 +298,25 @@ def py2and3_test(
         suite_kwargs["visibility"] = kwargs["visibility"]
 
     native.test_suite(
-        name = name,
+        name = name + ".both_pythons",
```

Comment:
optional: should we call it `pythons`, or add a TODO comment for removal?

Reply:
I prefer the latter. Tracking issue created.